ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/files/log_3scale_tenant_info.yml

```diff
@@ -29,21 +18,17 @@
     ocp4_workload_lpe_3scale_tenant_admin_url: "{{ r_dc.resources[0].data.adminURL | b64decode }}"
     ocp4_workload_lpe_3scale_tenant_access_token: "{{ r_dc.resources[0].data.token | b64decode }}"
 
-- name: Get the Tenant Portal Endpoint
-  set_fact:
-    ocp4_workload_lpe_3scale_tenant_portal_endpoint: "https://{{ ocp4_workload_lpe_3scale_tenant_access_token }}@{{ocp4_workload_lpe_3scale_tenant_admin_url | regex_search('(?<=https:\\/\\/).*') }}"
-
 - name: output tenant info
   agnosticd_user_info:
     msg: "{{ item }}"
   loop:
     - "user.info: "
```

Comment:
You don't need to prefix `user.info: ` if you use the agnosticd_user_info module.

Reply:
Done